checkov/dockerfile/checks/graph_checks/EnvGitSslNoVerify.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+  id: "CKV2_DOCKER_14"
+  name: "Ensure that certificate validation isn't disabled for git by setting the environment variable 'GIT_SSL_NO_VERIFY' to any value"
+  category: "APPLICATION_SECURITY"
+definition:
+  or:
+    - cond_type: attribute
+      resource_types:
+        - ARG
+        - ENV
+      attribute: value
+      operator: not_regex_match
+      value: "(.*\\s+)?(GIT_SSL_NO_VERIFY(=|\\s+)[^\\s]+).*"
```

Comment:
from the examples I have seen you would need to set it to `true`, maybe also `1` works, but this would also flag a case of
```docker
ENV GIT_SSL_NO_VERIFY false
```

Reply:
@james-otten-pan thanks for the insight, looks like the value doesn't matter, just the existence of the env var.